Validated_Partners/vSphere_Partners.txt

```diff
@@ -3,3 +3,4 @@
  suse
  newrelic
  hashicorp
+ sysdig
```

Comment:
Please remove this. We update this with automation once your changes are approved, tested and merged